sky/global_user_state.py

```diff
@@ -290,7 +428,9 @@ def get_clusters() -> List[Dict[str, Any]]:
             'autostop': autostop,
             'to_down': bool(to_down),
             'metadata': json.loads(metadata),
+            'cluster_hash': cluster_hash,
         }
```

Comment:
Can we include the `cost` in this dict as well? In that way, we can reduce the abstraction leakage in the `status_utils`. Also, it will be easier for the user to get the cost of the cluster with the programmatic API.